### ISSUE TITLE
SDK crashes if Discord isn't open

### ISSUE THREAD

**Issue by joaoburatto** [eslam980/Ultimate-Discord-SDK#3]
When launching a game build, if the Discord app isn't open, the game will crash and force open Discord.

**Comment by eslam980** [comment]
can you provide me with more info , and if you need any help just contact me.

**Comment by joaoburatto** [comment]
updated the description

**Comment by eslam980** [comment]
clever fix , I am working on A Big Update will release soon , and a Official Release on the Unity Assets Store soon too. and Thank you for using my SDK , if you need anything you can contact me or add me in discord " eslam980#9639 ";  .

**Comment by peq42** [comment]
issue is still happening.

**Comment by eslam980** [comment]
yeah I know , but I try to update it in this week if I can't in this week I will update it later after my exams.
